Sources/TuistLoader/Loaders/CachedManifestLoader.swift

```diff
@@ -1,6 +1,7 @@
 import Foundation
 import ProjectDescription
 import TSCBasic
+import TuistGraph
```

Comment:
alternatively, you could do `import struct TuistGraph.Plugins` if you did not want to prepend all the returned models with `ProjectDescription`. But the current solution is fine, too